src/Microsoft.AspNetCore.Routing/DefaultInlineConstraintResolver.cs

```diff
@@ -26,8 +25,14 @@ public class DefaultInlineConstraintResolver : IInlineConstraintResolver
         /// Accessor for <see cref="RouteOptions"/> containing the constraints of interest.
         /// </param>
         public DefaultInlineConstraintResolver(IOptions<RouteOptions> routeOptions)
```

Comment:
Should we obsolete this? If anyone is using it, we'd recommend that we use the other constructor now.

Reply:
Sure